Gems/AudioSystem/Code/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ ly_add_target(
     BUILD_DEPENDENCIES
         PRIVATE
             Legacy::CryCommon
+            AZ::AtomCore
```

Comment:
Wait, why does the Audio System need a dependency on Atom?

Reply:
Debug Draw is the reason.